documentation/md/collection/layoutPositions.md

```diff
@@ -0,0 +1,23 @@
+This function is called by a layout to update the graph with new node positions.
+
```

Comment:
May be worth mentioning that it's mostly for discrete layouts.  Continuous ones (e.g. force-directed) wouldn't use this except in cases like `cose-bilkent` `animate: end`.